modules/troubleshooting-dspa-component-errors.adoc

```diff
@@ -105,99 +111,120 @@ Ready
+
+Condition type: `FailingToDeploy`
 |False
-|`Component <component> has failed to progress. Reason: <progressingCond.Reason>. Message: <progressingCond.Message>`
-|The deployment has stalled due to `ProgressDeadlineExceeded` or `ReplicaSetCreateError` issues, or similar.
+|Error message: `Component <component> has failed to progress. Reason: <progressingCond.Reason>. Message: <progressingCond.Message>`
+
+Solution: The deployment has stalled due to `ProgressDeadlineExceeded` or `ReplicaSetCreateError` issues, or similar.
+
+|Condition: Replica Creation Failure
 
-|Replica Creation Failure
-|`FailingToDeploy`
+Condition type: `FailingToDeploy`
 |False
-|`Component's replica <component> has failed to create. Reason: <replicaFailureCond.Reason>. Message: <replicaFailureCond.Message>`
-|Replica creation has failed, typically due to an error in the replica set or with the service accounts.
+|Error message: `Component's replica <component> has failed to create. Reason: <replicaFailureCond.Reason>. Message: <replicaFailureCond.Message>`
 
-|Pod-Level Failures
-|`FailingToDeploy`
+Solution: Replica creation has failed, typically due to an error in the replica set or with the service accounts.
+
+|Condition: Pod-Level Failures
+
+Condition type: `FailingToDeploy`
 |False
-|`Concatenated failure messages for each pod.`
-|Deployment pods are in a failed state. Check the pod logs for further information.
+|Error message: `Concatenated failure messages for each pod.`
+
+Solution: Deployment pods are in a failed state. Check the pod logs for further information.
 
-|Pod in CrashLoopBackOff
-|`FailingToDeploy`
+|Condition: Pod in CrashLoopBackOff
+
+Condition type: `FailingToDeploy`
 |False
-|`Component <component> is in CrashLoopBackOff. Message from pod: <crashLoopBackOffMessage>`
-|Pod containers are failing repeatedly, often due to incorrect environment variables or missing service accounts.
+|Error message: `Component <component> is in CrashLoopBackOff. Message from pod: <crashLoopBackOffMessage>`
+
+Solution: Pod containers are failing repeatedly, often due to incorrect environment variables or missing service accounts.
+
+|Condition: Component Deploying (No Errors)
 
-|Component Deploying (No Errors)
-|`Deploying`
+Condition: type: `Deploying`
 |False
-|`Component <component> is deploying.`
-|The component deployment process is ongoing with no errors detected.
+|Error message: `Component <component> is deploying.`
 
-|Component Minimally Available
-|`MinimumReplicasAvailable`
+Solution: The component deployment process is ongoing with no errors detected.
+
+|Condition: Component Minimally Available
+
+Condition type: `MinimumReplicasAvailable`
 |True
-|`Component <component> is minimally available.`
-|The component is available, but only with the minimum number of replicas running.
+|Error message: `Component <component> is minimally available.`
+
+Solution: The component is available, but only with the minimum number of replicas running.
```

Comment:
Swap the word order to "with only". See the "Adverbs - only" entry in _IBM Style_.